### PR TITLE
[4.0] a11y improvement for admin toolbar

### PR DESCRIPTION
**Pull request by brianteeman**
This adds a role of toolbar and an aria-label of toolbar to the default admin toolbar used on most components.

(There may be other places that this needs to be applied as well which i did in another pr for 3.x and will hopefully be merged across but this one was definitely different and needed its own PR)

### CODE REVIEW

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on 10a36088e947006630b18945bf26f3d8c6937f5c<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15586">issues.joomla.org/tracker/joomla-cms/15586</a>.</sub>

**Comment by infograf768** [comment]
Is toolbar used in frontend?

**Comment by infograf768** [comment]
For frontend config?

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on 10a36088e947006630b18945bf26f3d8c6937f5c<br /><br />Based on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15586">issues.joomla.org/tracker/joomla-cms/15586</a>.</sub>

**Comment by C-Lodder** [comment]
I have not tested this item.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15586">issues.joomla.org/tracker/joomla-cms/15586</a>.</sub>

File: layouts/joomla/toolbar/containeropen.php [hunk 1]
```diff
@@ -9,4 +9,4 @@
 
 defined('JPATH_BASE') or die;
 ?>
-<div class="btn-toolbar d-flex" id="<?php echo $displayData['id']; ?>">
+<div class="btn-toolbar d-flex" role="toolbar" aria-label="<?php echo JText::_('JTOOLBAR') ?>"> id="<?php echo $displayData['id']; ?>">
```
Comment (C-Lodder) [inline layouts/joomla/toolbar/containeropen.php]: Semi-colon needed after `JText::_('JTOOLBAR')`

**Comment by brianteeman** [comment]
@infograf768 frontend config was handled in the other pr which will hopefully be merged for 3.x

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on 279117ddfd3494b95a1303ebc93493bb1f8c37e3<br /><br />Based on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15586">issues.joomla.org/tracker/joomla-cms/15586</a>.</sub>

**Comment by C-Lodder** [comment]
Can we RTC this?

**Comment by dgt41** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15586">issues.joomla.org/tracker/joomla-cms/15586</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**